pyravendb/store/document_session.py

```diff
@@ -19,6 +20,126 @@ def __init__(self, commands, deferred_command_count, entities=None):
         self.deferred_command_count = deferred_command_count
 
 
+class _RefEq:
+    def __init__(self, ref):
+        if isinstance(ref, _RefEq):
+            self.ref = ref.ref
+        self.ref = ref
+
+    def __eq__(self, other):
+        return (
+            True
+            if id(self.ref) == id(other)
+            else False
```

Comment:
Can we put another set of parens here for clarity? 👓 Or maybe do it a bit more verbose using regular if/else.

Reply:
simplified in latest commit